components/rule-mgt/org.wso2.carbon.identity.rule.management/pom.xml

```diff
@@ -131,6 +129,8 @@
                             version="${org.wso2.carbon.database.utils.version.range}",
                             org.wso2.carbon.identity.core.cache;
                             version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.core.util;
```

Comment:
util package is usually considered as an internal package right ? In that case can't we move `RuleBuilder` to the model package ?

Reply:
It depends on what's there in the package.
This util needs to be exposed